chart/templates/cleanup/cleanup-cronjob.yaml

```diff
@@ -61,7 +61,7 @@ spec:
               image: {{ template "default_airflow_image" . }}
               imagePullPolicy: {{ .Values.images.airflow.pullPolicy }}
               # Don't use entry point here, we don't need to wait on pg-bouncer etc being available.
-              command: ["airflow-cleanup-pods", "--namespace={{ .Release.Namespace }}"]
+              args: ["kubernetes", "cleanup-pods", "--namespace={{ .Release.Namespace }}"]
```

Comment:
One comment left: https://github.com/apache/airflow/pull/11802#issuecomment-715665276

Reply:
Yes I am planning to backport this, will add 1.10.13 milestone